### DISCUSSION TITLE
MAA配置排序突然变得混乱了…？

### DISCUSSION THREAD

**Discussion by TimCrownie** [MaaAssistantArknights/MaaAssistantArknights#10633] [General]
![image](https://github.com/user-attachments/assets/03197cfa-148e-473f-a364-bb789a9aa6e4)
![image](https://github.com/user-attachments/assets/cb340b71-9b6a-403e-b75d-1cdfb380f49f)
加了数字也没什么用啊…
（因为觉得没到要发issue的程度所以就发discussion了
原本的顺序是
α-EN α β γ Mi5-EN XZ1 Mi5 wawei
现在就是全乱掉了嘛…

**Comment by ABA2396** [comment]
这玩意本来就没做排序，直接从 Dictionary 转的 List ，之后加个排序好了

**Reply by TimCrownie** [reply]
好好好（

**Reply by ABA2396** [reply]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/82a5f53ff13acc99cb6e7a240412ad028ba09eb6 fixed
